the-collab-lab/tcl-16-smart-shopping-list
# As a user, I want the app to look professional and welcoming

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description


We updated the code to ensure the shopping list screen, welcome screen, and add item screen look professional and welcoming. We chose to use sass to style our app so that we have more control of the appearance and took advantages of mixins, extends, and variables for clean/reusable code. We also updated some of our JSX to improve layout, semantic elements, stylistic changes, and accessibility. Furthermore, sweet alerts are now being implemented throughout our app to give it a more professional feel. 

## Related Issue


closes #13 

## Acceptance Criteria


The welcome screen, add item screen, and shopping list screen must look professional and visually appealing. 

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|  ✓ | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before



![](https://user-images.githubusercontent.com/46036289/109092514-abb02180-76e4-11eb-81c2-c254d83879af.png)

![](https://user-images.githubusercontent.com/46036289/109238597-cf856d00-77a1-11eb-8246-0f3fc1dd30cd.png)

![68747470733a2f2f692e6962622e636f2f484b48374a6b422f61667465722d6e61766261722e706e67](https://user-images.githubusercontent.com/46036289/109421789-cca79980-79a6-11eb-90a6-b5f9a70f0e86.png)

### After



![](https://user-images.githubusercontent.com/46036289/109092597-d69a7580-76e4-11eb-9476-5f5d9b3b4ceb.png)

![](https://user-images.githubusercontent.com/46036289/109238622-dca25c00-77a1-11eb-80f2-846d85da0122.png)

![68747470733a2f2f692e6962622e636f2f78713931537a542f53637265656e73686f742d323032312d30322d32372d3136333234312e706e67](https://user-images.githubusercontent.com/46036289/109421811-e5b04a80-79a6-11eb-8a32-e733fbccd6d4.png)

## Testing Steps / QA Criteria


1. switch to branch `issue-13-development` and `git pull` to ensure you have latest commits
2. to view the welcome screen, ensure your localStorage is cleared 
3. to view add item screen, click on the '+' icon in the navbar which is on the top of the page 
4. to view shopping list screen, click on the list icon in the navbar which is on the top of the page (ensure you have added some items if your shopping list is empty)
